google/cloud/exceptions.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, message, errors=()):
         self._errors = errors
 
     def __str__(self):
-        return '%d %s' % (self.code, self.message)
+        return u'%d %s' % (self.code, self.message)
```

Comment:
Unfortunately an `if six.PY2` might be necessary here. Though it's not that sad since `__str__` doesn't need to be invoked during performant code.

```python
result = u'%d %s' % (self.code, self.message)
if six.PY2
    result = _helpers._to_bytes(result)
return result
```